### PR TITLE
Rename package to pynear and upload to pypi on github release

### PR DESCRIPTION
**Pull request by Dobatymo**
It's a lot of file renames since the top level dir changed from pyvptree to pynear

### CODE REVIEW

**Comment by pablocael** [comment]
> It's a lot of file renames since the top level dir changed from pyvptree to pynear

Its expected no prob. I just want to maybe merge the other PRs first, not sure if will be same or easier to merge first.

**Comment by pablocael** [comment]
I think here: https://github.com/pablocael/pynear/pull/35/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5
Since our goal is to not exclusively focus on VPTrees but rather use it as one of available algorithms, we can change to something like "pynear implements a set of algorithms for searching nearest neighbors, among them the VPTree structure" and then talk about the VPTree. Or we can move the VPTree explanation to a link within docs/README.md where I list the indices in a table. We could put a link for thre indice and explain it there. 

Maybe I will do this after basic migration to pynear in a separate PR.

File: README.md [hunk 1]
```diff
@@ -45,10 +45,10 @@ For more features and all available index types, refer to [docs](./docs/README.m
 
 ## Pickle serialization
 
-Pyvptree is pickle serializable:
+vptree indices is pickle serializable:
```
Comment (pablocael) [inline README.md:48]: is -> are

**Comment by Dobatymo** [comment]
> I think here: https://github.com/pablocael/pynear/pull/35/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5 Since our goal is to not exclusively focus on VPTrees but rather use it as one of available algorithms, we can change to something like "pynear implements a set of algorithms for searching nearest neighbors, among them the VPTree structure" and then talk about the VPTree. Or we can move the VPTree explanation to a link within docs/README.md where I list the indices in a table. We could put a link for thre indice and explain it there.
> 
> Maybe I will do this after basic migration to pynear in a separate PR.

Yes if you don't mind we modify the readme in a separate PR. This one changes so many files already.

**Comment by pablocael** [comment]
I think we need to change `Makefile` `make test` command. I tried your branch here and test is passing by make file target needs to be fixed to pynear. Other than that is fine. Great work. thanks a lot! @Dobatymo

**Review by pablocael — APPROVED** [review]

**Pull request merged**